src/gdclasses/OpenXRConfig.h

```diff
@@ -30,6 +30,10 @@ class OpenXRConfig : public Node {
 	int get_form_factor() const;
 	void set_form_factor(const int p_form_factor);
 
+	double get_refresh_rate() const;
+	void set_refresh_rate(const double p_refresh_rate);
```

Comment:
Just in case anyone asks, I'm using doubles here even though floats are used in the API. This is because with the new GD extensions everything is converted to doubles, might as well start using that as a base.